File: JavaScript/Iniciante/1013.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

linha = lines.shift().split(" ")
a = parseInt(linha[0])
b = parseInt(linha[1])
c = parseInt(linha[2])

maiorAB = (a + b + Math.abs(a - b)) / 2
maiorABC = (maiorAB + c + Math.abs(maiorAB - c)) / 2

console.log(`${maiorABC} eh o maior`);

File: Elixir/Iniciante/1013.exs
valores = IO.gets("") |> String.trim()
valor = String.split(valores, " ") |> Enum.map(&String.trim/1)

a = Enum.at(valor, 0) |> String.to_integer
b = Enum.at(valor, 1) |> String.to_integer
c = Enum.at(valor, 2) |> String.to_integer

maiorAB = (a + b + abs(a-b)) / 2
maiorABC = (maiorAB + c + abs(maiorAB-c)) / 2

IO.puts "#{:erlang.float_to_binary(maiorABC, decimals: 0)} eh o maior"
